content/components/flash.mdx

```diff
@@ -4,6 +4,7 @@ description: Flash alert informs users of successful or pending actions.
 reactId: flash
 railsIds:
 - Primer::Beta::Flash
+- Primer::Alpha::Banner
 figmaId: banner
 rails: https://primer.style/view-components/components/beta/flash
```

Comment:
What's the difference between the `rails` property and the `railsIds` property?

Reply:
good question - /cc @camertron ?

Reply:
Oh, I think the `rails` property is really old. We can get rid of it.